portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path

```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+<title>MyPath</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MyPath</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>APPLICATIONS_MENU</td>
+	<td>//div[contains(@class,'control-menu')]//button[@data-qa-id='applicationsMenu']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CONTROL_PANEL</td>
+	<td>//li[@class='nav-item']/button[child::span[text()='Control Panel']]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PANEL_CONFIGURATION_SECTION</td>
+	<td>//div[@class='col-md' and child::nav[@aria-labelledby='nav_control_panel.configuration']]</td>
+	<td></td>
```

Comment:
Hey, you can use this xpath //h2[@id='nav_control_panel.configuration'], like I explained in this [PR](https://github.com/manuferreira/liferay-portal/pull/72/files#r1313168549) before, thank you!

Reply:
Alright. Thank you!